amivapi/events/model.py

```diff
@@ -181,7 +182,7 @@
             },
             'additional_fields': {
                 'nullable': True,
-                'type': 'json_schema',
+                'type': 'cerberus_schema',
```

Comment:
no it is just a name change. Look here https://github.com/amiv-eth/amivapi/pull/159/commits/47ec1aa0cda86cf0c93bc104f396948d3823e622#diff-95bf83f7b01c7278d2a3d182af15cc19R145
The name json_schema was misleading, because it IS a cerberus schema we are parsing.